### PR TITLE
Add endroid/pdf recipe

### PR DESCRIPTION
**Pull request by endroid**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: endroid/pdf/1.0/config/packages/endroid_pdf.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    Endroid\Pdf\Pdf: ~
```
Comment (Nyholm) [inline endroid/pdf/1.0/config/packages/endroid_pdf.yaml]: Do we really need to auto wire just one service? The config will be smaller if we don’t :)

Comment (endroid) [reply endroid/pdf/1.0/config/packages/endroid_pdf.yaml]: Yes this is the only service that needs to be wired. How would you make it shorter? If I leave out line 7 the service is not found and if I leave out the _defaults the constructor arguments are not injected automatically. But if you think there is a part we can leave out while still having an autowired service, I'll be happy to remove that.

Comment (Nyholm) [reply endroid/pdf/1.0/config/packages/endroid_pdf.yaml]: How about this: 
```
services: 
    Endroid\Pdf\Pdf: 
        autowire: true
```

??

Comment (endroid) [reply endroid/pdf/1.0/config/packages/endroid_pdf.yaml]: Ah that's what you meant. I personally like to keep autowiring in the defaults in all my projects. I agree with you that it is not really necessary here, but it is a matter of personal preference.

Comment (Nyholm) [reply endroid/pdf/1.0/config/packages/endroid_pdf.yaml]: Yes it is. So lets be consistent with all the other and make the service declaration easier to read. Then I'll be happy to approve this PR.

Comment (endroid) [reply endroid/pdf/1.0/config/packages/endroid_pdf.yaml]: @Nyholm I tried the configuration as above but then Travis throws an error that Knp\Snappy\Pdf (which should be wired as the first argument) is not found. When I change the service definition back to the one with the _defaults key the error is gone again.

Do you have any idea why the first configuration works and the latter not?

**Pull request merged**